schema/mysql/v57/temporal/schema.sql

```diff
@@ -91,15 +91,13 @@ CREATE TABLE tasks (
 );
 
 CREATE TABLE task_queues (
-  shard_id INT NOT NULL,
-  namespace_id BINARY(16) NOT NULL,
-  name VARCHAR(255) NOT NULL,
-  task_type TINYINT NOT NULL, -- {Activity, Workflow}
+  range_hash INT UNSIGNED NOT NULL,
+  task_queue_id VARBINARY(255) NOT NULL,
```

Comment:
Do you want to change the size as this is now a concatenation of multiple fields.

Reply:
Will follow up on this with additional change where I add indexes as well.